### PR TITLE
Make top bar button white for accessibility reasons

### PR DESCRIPTION
**Pull request by petesfrench**
Design team 'Public'/'Private' button in the top bar be white to match the rest of the other buttons for accessibility reasons.
https://warthogs.atlassian.net/browse/WD-5291

### CODE REVIEW

**Comment by petesfrench** [comment]
@carkod Could you look at this one also please?

**Comment by carkod** [comment]
> @carkod Could you look at this one also please?

How do I QA this?

**Review by carkod — APPROVED** [review]
Code looks good. But haven't been able to QA

File: css/00-main.css [hunk 1]
```diff
@@ -130,18 +130,13 @@ ul.category-list li {
...
-     border-radius: 2px 0 0 2px;
-     background: #E95420;
-     border-color: #E95420;
-     color: #ffffff !important;
+.i-button.accept:not(.label):not(.borderless):not(.text-color):hover, div.session-bar .i-button.protection-public:not(.label):not(.borderless):not(.text-color):hover, div.session-bar .protection-public.action-button:not(.label):not(.borderless):not(.text-color):hover, .accept.action-button:not(.label):not(.borderless):not(.text-color):hover {
+    background: #fff !important;
+    color: #111 !important;
+}
+.i-button.accept:not(.label):not(.borderless):not(.text-color), div.session-bar .i-button.protection-public:not(.label):not(.borderless):not(.text-color), div.session-bar .protection-public.action-button:not(.label):not(.borderless):not(.text-color), .accept.action-button:not(.label):not(.borderless):not(.text-color) {
+    background: #fff;
```
Comment (carkod) [inline css/00-main.css:138]: Maybe should be uppercase to follow the same styling?

**Comment by kewisch** [comment]
image

Is this the intended look? I just tested this in the style editor, so maybe something is wrong. Do you have more background why a white button will increase accessibility? Is this about contrast ratio?

Design may have a better expert opinion, but I do prefer the additional color the orange button brings in. Note also that some event themes overwrite the button color on event pages, e.g. the 'Canonical' theme makes it aubergine.

**Comment by petesfrench** [comment]
@kewisch I wasn't aware there were differences across different themes. I would image this would also be in-accessible. I would suggest we keep the navigation in the default colours in this case. But I will bring this up with design first and see if they are willing to compromise.

**Comment by petesfrench** [comment]
Closing as this work is no longer needed.

**Pull request closed without merging**